projects/browser-phase-1.md

```diff
@@ -90,15 +90,20 @@ Options include:
 
 ## Staffing / Help Wanted
 
+### Industry Outreach
+
+There are a number of existing RUM/Client observability implementations. Representatives from a wide selection of companies that have experience with browser observability are present in the SIG. This includes Microsoft, New Relic, DynaTrace, HoneyComb, Grafana Labs, and Cisco, among others. We plan for further outreach among end users once we have working code for them to review.
+
 ### Required Staffing
 
 **Project Lead:** Ted Young
 
 **GC Liaison:** Daniel Gomez Blanco
 
 **Sponsors:**
-* Ted Young
-* SPONSOR NEEDED
+* Ted Young - Spec Maintainer Sponsor
+* Daniel Dyla - JS Maintainer Sponsor
+* Carlos Alberto Cortez – TC Escalating Sponsor
 
 **Implementation Engineers:**
```

Comment:
I understand that Martin is from ServiceNow and Purvi from Honeycomb. Didn't the other companies in your list commit to help in the implementation? I see Marco (Grafana) and David (Elastic) in the list of NodeJS maintainers and approvers here (apparently, it doesn't match with the list I see on opentelemetry-js?): are they also committing to work on the implementation?

Reply:
Ok, I can review the implementation engineers section with the client SIG and update it with more commitments.

I'm not sure what you mean about NodeJS approvers section? Everyone listed in that section is part of the javascirpt approvers team. Can you clarify?